src/content/docs/design/design-with-code/about-designing-with-code.mdx

```diff
@@ -0,0 +1,32 @@
+---
+page_id: 66f83c18-d261-48b8-a517-648ed1b7b064
+title: About designing with code
```

Comment:
@clairekinde11 `designing with code` seems like a strange way of phrasing it. Probably customizing with code?